test/Microsoft.Health.Fhir.Tests.E2E/Common/FhirClientExtensions.cs

```diff
@@ -61,12 +61,18 @@ public static async Task<TResource> CreateResourcesAsync<TResource>(this FhirCli
 
         public static async Task<TResource[]> CreateResourcesAsync<TResource>(this FhirClient client, params Action<TResource>[] resourceCustomizer)
             where TResource : Resource, new()
+        {
+            return await CreateResourcesAsync(client, () => new TResource(), resourceCustomizer);
+        }
+
+        public static async Task<TResource[]> CreateResourcesAsync<TResource>(this FhirClient client, Func<TResource> setup, params Action<TResource>[] resourceCustomizer)
```

Comment:
Can you also update the rest of the text fixture to follow the same pattern? (e.g., NumberSearchTestFixture still uses the old way).

Reply:
This should now be aligned, I used the existing local function pattern